sonar-project.properties

```diff
@@ -7,19 +7,39 @@ sonar.typescript.tsconfigPath=tsconfig.json
 sonar.eslint.reportPaths=eslint-report.json
 
 # Path to sources
-sonar.sources=benchmark,ci,desktop,packages,web
-sonar.exclusions=**/*.test.ts,**/*.test.tsx,**/integration-test/**
+sonar.sources=\
+  benchmark,\
+  ci,\
+  desktop,\
+  packages,\
+  web
+sonar.exclusions=\
+  **/*.test.ts,\
+  **/*.test.tsx
```

Comment:
it is on purpose to remove the **/integration-test/** from here?

Reply:
I've just discovered a proper configuration not to consider the integration tests. Then, I moved the integration-tests folder to "sonar.coverage.exclusions".